atmo/keys/models.py

```diff
@@ -56,6 +56,5 @@ def prefix(self):
         return self.key.strip().split()[0]
 
     def save(self, *args, **kwargs):
-        if not self.fingerprint and self.key:
-            self.fingerprint = calculate_fingerprint(self.key)
+        self.fingerprint = calculate_fingerprint(self.key)
```

Comment:
Yeah, the commit tests the behavior, which we change here of always updating the fingerprint based on the key value, https://github.com/mozilla/telemetry-analysis-service/pull/173/files#diff-da6b35707e8a7f55bbf2ba043124f462R61

The reasoning here is that we don't want to accidentally update the key somehow and not be able to verify the fingerprint separately. In other words, there is no reason to have the fingerprint calculation happen conditionally and only on the first save.